src/main/java/task/core/TaskManager.java

```diff
@@ -30,19 +35,30 @@
 
     private IJobRecordConnection monitorConnection;
 
+    /**
+     * Constructor for a TaskManager object
+     */
     private TaskManager() {
         monitorConnection = JobRecordConnectionProvider.connection();
     }
 
+    /**
+     * Method that returns an instance of a TaskManager
+     * @return - an instance of a TaskManager object
+     */
     public static TaskManager instance() {
         if (instance == null) {
             instance = new TaskManager();
         }
         return instance;
     }
 
+    /**
+     * Method that sends tasks
```

Comment:
Indicate what is the period, what type of tasks are created. Mention that tasks are created in background (not in main thread). Also add TODO about creating a method to stop this process. Instead of having inline comment about functionality of methods, add them to top of function. Inline comments are good if Code does some confusing/unexpected operation.